src/components/textarea/textarea.pw.tsx

```diff
@@ -686,6 +686,34 @@ test.describe("Props tests for Textarea component", () => {
   });
 });
 
+test(`should verify expandable Textarea shrinks back to original height when lines are removed`, async ({
+  mount,
+  page,
+}) => {
+  await mount(<TextareaComponent expandable />);
+
+  const textareaChildrenElement = textareaChildren(page);
+  await textareaChildrenElement.type("t");
+  await textareaChildrenElement.press("Enter");
+  await textareaChildrenElement.type("e");
+  await textareaChildrenElement.press("Enter");
+  await textareaChildrenElement.type("s");
+  await textareaChildrenElement.press("Enter");
+  await textareaChildrenElement.type("t");
```

Comment:
**suggestion**: it appears [the Locator method `type` has been deprecated by Playwright](https://playwright.dev/docs/api/class-locator#locator-type), what do you think about replacing it with `press` to work around this?: 

```suggestion
  await textareaChildrenElement.press("t");
  await textareaChildrenElement.press("Enter");
  await textareaChildrenElement.press("e");
  await textareaChildrenElement.press("Enter");
  await textareaChildrenElement.press("s");
  await textareaChildrenElement.press("Enter");
  await textareaChildrenElement.press("t");
```